src/instana/instrumentation/psycopg2.py

```diff
@@ -5,33 +5,44 @@
 import copy
 import wrapt
 
-from ..log import logger
-from .pep0249 import ConnectionFactory
+from typing import Callable, Optional, Any, Tuple, Dict
+from instana.log import logger
+from instana.instrumentation.pep0249 import ConnectionFactory
 
 try:
     import psycopg2
     import psycopg2.extras
 
-    cf = ConnectionFactory(connect_func=psycopg2.connect, module_name='postgres')
+    cf = ConnectionFactory(connect_func=psycopg2.connect, module_name="postgres")
 
-    setattr(psycopg2, 'connect', cf)
-    if hasattr(psycopg2, 'Connect'):
-        setattr(psycopg2, 'Connect', cf)
+    setattr(psycopg2, "connect", cf)
+    if hasattr(psycopg2, "Connect"):
+        setattr(psycopg2, "Connect", cf)
 
-    @wrapt.patch_function_wrapper('psycopg2.extensions', 'register_type')
-    def register_type_with_instana(wrapped, instance, args, kwargs):
+    @wrapt.patch_function_wrapper("psycopg2.extensions", "register_type")
+    def register_type_with_instana(
+        wrapped: Callable[..., Any],
+        instance: Optional[Any],
+        args: Tuple[Any, ...],
+        kwargs: Dict[str, Any],
+    ) -> Callable[..., Any]:
```

Comment:
Can we be more specific about the return type of the callable? If its too complicated, you can change it to below.

```suggestion
        wrapped: Callable[..., object],
        instance: Optional[Any],
        args: Tuple[Any, ...],
        kwargs: Dict[str, Any],
    ) -> Callable[..., object]:
```

Reply:
Done